### ISSUE TITLE
Functions not executed in custom properties

### ISSUE THREAD

**Issue by apfelbox** [sass/libsass#2629]
Hi,

it seems, that color functions (or any function if I see it correctly) are not compiled when using custom properties.

## input.scss

[todo]: # (always test and report with scss syntax)
[todo]: # (use sass only when results differ from scss)

```scss
$color: #0f0;

.button {
    color: rgba($color, .2);
    --color: rgba($color, .2);
}

```

## Actual results

[libsass 3.5.2][1]

```css
.button {
  color: rgba(0, 255, 0, 0.2);
  --color: rgba($color, .2); }
```

## Expected result

ruby sass 3.X.y
```css
.button {
  color: rgba(0, 255, 0, 0.2);
  --color: rgba(0, 255, 0, 0.2); }
```

version info:
```cmd
$ sassc -v
sassc: 3.4.8
libsass: 3.5.2
sass2scss: 1.1.1
sass: 3.5
```

[todo]: # (Go to http://libsass.ocbnet.ch/srcmap)
[todo]: # (Enter your SCSS code and hit compile)
[todo]: # (Click `bookmark` and replace the url)
[todo]: # (link is used in actual results above)

[1]: http://libsass.ocbnet.ch/srcmap/#JGNvbG9yOiAjMGYwOwoKCi5idXR0b24gewogICAgY29sb3I6IHJnYmEoJGNvbG9yLCAuMik7CiAgICAtLWNvbG9yOiByZ2JhKCRjb2xvciwgLjIpOwp9Cg==

**Comment by xzyfer** [comment]
```
--color: #{rgba($color, .2)};
```

**Comment by apfelbox** [comment]
I know of this workaround, but is this not a bug? @xzyfer

**Comment by apfelbox** [comment]
(Sassmeister tells me, that this might be fixed in LibSass 3.5.0.beta.2 https://www.sassmeister.com/gist/62baf195a564f3ac62a954a03d4f8586)

**Comment by nschonni** [comment]
Duplicate of #2585
